### PR TITLE
FindSurfaceFeaturesTest: Use downloaded files, remove test files not needed

### PR DESCRIPTION
**Pull request by imikejackson**

### CODE REVIEW

File: src/Plugins/ComplexCore/src/ComplexCore/Filters/CreateDataArray.cpp [hunk 1]
```diff
@@ -73,7 +73,6 @@ Parameters CreateDataArray::parameters() const
 
   params.insert(std::make_unique<NumericTypeParameter>(k_NumericType_Key, "Numeric Type", "Numeric Type of data to create", NumericType::int32));
   params.insert(std::make_unique<StringParameter>(k_InitilizationValue_Key, "Initialization Value", "This value will be used to fill the new array", "0"));
-  params.insert(std::make_unique<ArrayCreationParameter>(k_DataPath_Key, "Created Array", "Array storing the data", DataPath{}));
```
Comment (nyoungbq) [inline src/Plugins/ComplexCore/src/ComplexCore/Filters/CreateDataArray.cpp:76]: Why is this parameter being removed?

**Comment by nyoungbq** [comment]
@imikejackson The `CreateDataArray` duplicate parameter patch was already fixed on develop: https://github.com/BlueQuartzSoftware/complex/commit/dc262f616ee12d429cdd7272cd5901cfe1ca89b4

It looks like you overwrote the fixes I made in this commit: https://github.com/BlueQuartzSoftware/complex/commit/5d79db038a446344156cd585969942f10cd97df9#diff-13d2df9518b71825e9d40caea9dc581541a8ee1e25040d3d1ae8494e3d2fa1ee

File: src/Plugins/ComplexCore/src/ComplexCore/Filters/CreateDataArray.cpp (hunk 1 above)
Comment (imikejackson) [reply src/Plugins/ComplexCore/src/ComplexCore/Filters/CreateDataArray.cpp:76]: It should be lower in the code

Comment (imikejackson) [reply src/Plugins/ComplexCore/src/ComplexCore/Filters/CreateDataArray.cpp:76]: On line 85.

**Review by nyoungbq — APPROVED** [review]

**Review by JDuffeyBQ — APPROVED** [review]

**Review by nyoungbq — APPROVED** [review]

**Pull request merged**